presto-docs/src/main/sphinx/release/release-328.rst

```diff
@@ -0,0 +1,63 @@
+========================
+Release 328 (6 Jan 2020)
+========================
+
+General Changes
+---------------
+
+* Fix correctness issue for certain correlated JOIN queries when the subquery on the
+  right produces no rows. (:issue:`1969`)
```

Comment:
Should this be "when the correlated subquery produces no rows" instead of "on the right"?

Reply:
That's not precise enough, as the left could be correlated on something else. The issue happens when the right side is correlated against the left side and produces no rows.